server/src/pages/settings/controller.js

```diff
@@ -20,7 +20,8 @@ exports.disconnectDevice = function() {
       window.alert("Error disconnecting: " + req.status + " " + req.statusText);
       window.Raven.captureException(new Error(`Error disconnecting: ${req.status} ${req.statusText}`));
     } else {
-      location.reload();
+      // Redirect to homepage
+      window.location = "/";
```

Comment:
Why this change?  And would My Shots make more sense than the home page since we know this is a Screenshots user?